wagtail/documents/models.py

```diff
@@ -23,7 +24,10 @@ class DocumentQuerySet(SearchableQuerySetMixin, models.QuerySet):
 
 class AbstractDocument(CollectionMember, index.Indexed, models.Model):
     title = models.CharField(max_length=255, verbose_name=_('title'))
-    file = models.FileField(upload_to='documents', verbose_name=_('file'))
+    file = models.FileField(
+        upload_to="documents",
+        verbose_name=_("file")
+    )
```

Comment:
This does not change anything, does it?

Reply:
Yes, this doesn't change everything. My bad it got reformatted in the adding and removing validators. 
Should I change it back to the formatting it was before or this is okay?

Reply:
Yes please. 🙇